internal/controller/components/kueue/kueue_controller_actions.go

```diff
@@ -38,12 +41,14 @@ func checkPreConditions(ctx context.Context, rr *odhtypes.ReconciliationRequest)
 
 func initialize(_ context.Context, rr *odhtypes.ReconciliationRequest) error {
 	rr.Manifests = append(rr.Manifests, manifestsPath())
+	rr.Manifests = append(rr.Manifests, kueueConfigManifestsPath())
```

Comment:
is the idea to have both
- /opt/manifests/kueue/<everything-as-is>
- /opt/manifests/kueue-config/<role + rolebding + 4.17> 
?
but then everything under` /opt/manifests/kueue-config` is a subset from `"/opt/manifests/kueue"`

Reply:
ok, then i need with https://github.com/opendatahub-io/kueue/pull/85 in, this wont be problem